development/banner_server.py

```diff
@@ -45,13 +45,20 @@ def run(self):
                     # If there's no linux symbols, don't cry about it
                     pass
 
-            new_banners = banner_cache.read_new_banners(context, 'BannerServer', potentials, banner_cache.symbol_name,
-                                                        banner_cache.os, progress_callback = PrintedProgress())
+            new_banners = banner_cache.read_new_banners(context,
```

Comment:
There's no need to keep the `development` directory to the same standards, but it also doesn't hurt.  5:)